### PR TITLE
Python 3 compatibility

### PR DESCRIPTION
**Pull request by travcunn**
Adds support for Python 3.

In the future, requests should be mocked using a library like "responses" or similar.

It may be good to run integration tests in Travis CI with environment variables to use someone's private Kloudless account.

### CODE REVIEW

**Comment by david-thorman** [comment]
This looks really great! We really appreciate the contribution to supporting Python3! There was just one small comment to address and then I will merge this in.

File: kloudless/util.py [hunk 1]
```diff
@@ -12,17 +12,14 @@ def to_datetime(timestamp):
...
-                        timestamp)
+
+    return dateutil.parser.parse(timestamp)
 
 def to_iso(obj):
     """
     Converts datetime object to an ISO 8601 timestamp.
     """
-    if isinstance(obj, basestring) or obj is None:
+    if isinstance(obj, str) or obj is None:
```
Comment (travcunn) [inline kloudless/util.py]: Good idea

**Comment by david-thorman** [comment]
These changes look good! I think that taking advantage of a library like `responses` would be good for the future. We just haven't had much time to devote to improvements here.

**Comment by vinodc** [comment]
Fantastic! I appreciate the improvements here.

**Pull request merged**